<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeEditor</class>
 <widget class="QWidget" name="NodeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>147</y>
       <width>169</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>303</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="4">
       <widget class="QPushButton" name="sendColorButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>images/send.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="colorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="onoffLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OnOff</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="nodeComboBox"/>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="sendOnOffButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>images/send.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="onoffCheckBox">
        <property name="text">
         <string>On</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="sendLevelButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>images/send.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ColorButton" name="colorButton">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="levelSpinBox"/>
      </item>
      <item row="2" column="2" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="levelLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QPushButton" name="sendLocationButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/send.png</normaloff>images/send.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="refreshDevicesButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>images/refresh.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Location</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>20</y>
           <width>251</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="northLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>North</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="northSpinBox">
            <property name="minimum">
             <number>-32768</number>
            </property>
            <property name="maximum">
             <number>32767</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="eastLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>East</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="eastSpinBox">
            <property name="minimum">
             <number>-32768</number>
            </property>
            <property name="maximum">
             <number>32767</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="altitudeSpinBox">
            <property name="minimum">
             <number>-32768</number>
            </property>
            <property name="maximum">
             <number>32767</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="altitudeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Altitude</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="floorLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Floor</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="floorSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="uncertaintyLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Uncertainty</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="uncertaintySpinBox">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
